vertx-mongo-client/src/main/java/io/vertx/ext/mongo/impl/codec/json/JsonObjectCodec.java

```diff
@@ -106,6 +149,14 @@ protected Object readObjectId(BsonReader reader, DecoderContext ctx) {
 
   @Override
   protected Object readDateTime(BsonReader reader, DecoderContext ctx) {
```

Comment:
I'm confused... Where does it say in the mongo docs that Mongo client supports:

{"$date": <epochMillis>}}

?

As far as I can tell it says it supports Mongo extended JSON which does not use this format,